### ISSUE TITLE
Hotkey issue in Dutch "Search & Replace" window.

### ISSUE THREAD

**Issue by ThelsK** [notepad-plus-plus/notepad-plus-plus#5587]
### Description of the Issue
In the Dutch version of the Replace panel (called Vervangen in Dutch), the hotkey for Replace All (called Alle Vervangen in Dutch) doesn't work. The capital A in "Alle Vervangen" is underlined, indicating that Alt-A should trigger that button. It doesn't. In fact, nothing seems to happen at all. When I manually use the mouse to click the button, it replaces as normal.

### Steps to Reproduce the Issue
1. Install the Dutch version of Notepad++.
2. Start a new document.
3. Write "Alice" in the document.
4. Press Ctrl-H to open the Replace panel.
5. Enter "Alice" for "Zoeken naar".
6. Enter "Bob" for "Vervangen door".
7. Press Alt-A to activate "Alle Vervangen".

### Expected Behavior
This should replace Alice by Bob when you activate "Alle Vervangen". The A in "Alle Vervangen" is underlined, so pressing Alt-A to trigger it should replace Alice with Bob,

### Actual Behavior
Nothing happens whatsoever.

### Debug Information
When using the mouse to click the "Alle Vervangen" button, Alice is properly replaced by Bob.

**Comment by th-lamb** [comment]
This happens in other languages as well.

I have accidently installed Notepad++ in German and switched the language to English manually. It looks like the shortcuts get confused by this?

It shows "Replace" with shortcut "R" but when i press CTRL+R i get asked whether i want to replace all occurrences. And nothing happens even if i answer "OK".

**Comment by xylographe** [comment]
@ThelsK 
I cannot reproduce. Both **Alt-E** (button "Vervangen" [eng:"Replace"]) and **Alt-A** (button "Alle vervangen" [eng:"Replace All"]) work as expected. Did you by any chance assign the **Alt-A** shortcut to some command or macro in the **Sneltoetsen Mapper**?

**Comment by ThelsK** [comment]
@Xylographe
I never did anything in there. I also had the issue on both my Desktop and my Laptop. I since then switched back to English on the next update, so I'm currently unable to properly test it. I'll see about installing the Dutch version on the next update to see if it's still an issue.

**Comment by Ekopalypse** [comment]
I remember about a discussion that it should be avoided to use the same shortcut for different tabs in the find dialog. @ThelsK, could it be that your dutch.xml contains multiple `&amp:A` in the dialog find tag section?

**Comment by xylographe** [comment]
@Ekopalypse 
Actually, the Alt-A keyboard accelerator is used in _two_ buttons in the Replace tab. Because the "Replace All" button comes first in the order of controls, the behaviour is as expected — but only by chance. :-D

I have added a correction to PR #5818.

**Comment by xylographe** [comment]
@ThelsK 
**Alt-E** ("Vervangen") and **Alt-A** ("Alle vervangen") work if (and only if) the cursor is already on the "Zoeken naar:" text. Hence you might have to press **Alt-V** (or click "Volgende zoeken") first. Perhaps that was the original problem?

@Ekopalypse 

> …it should be avoided to use the same shortcut for different tabs in the find dialog.

I read that dicussion too! But apparently that problem has been fixed somewhere between 7.5 and 7.7. The (updated) Dutch translation uses **Alt-B** as a button keyboard accelerator in the Search tab and the Replace tab, and **Alt-A** in the Replace tab and the Mark tab. Both key combinations work as expected in NPP 7.7.1.

**Comment by Ekopalypse** [comment]
@xylographe, strange, doesn't work for me and if I check dutch.xml from the repo
I see [here](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/dutch.xml#L362) and [here](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/dutch.xml#L357) and [here](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/nativeLang/dutch.xml#L367) multiple A-shortcuts. 
The only one which is working for me is the one assigned to `Alle zoeken`

**Comment by xylographe** [comment]
I am using the [updated dutch.xml](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/d3e84652e97310cccbac04a770e63d058f6719d9/PowerEditor/installer/nativeLang/dutch.xml).

Did you read what I wrote about positioning the cursor with **Alt-V** ("Volgende zoeken"/"Find Next") in the Replace tab, _before_ using **Alt-A** ("Replace All") or **Alt-E** ("Replace")?

**Comment by Ekopalypse** [comment]
Yes, I read but does not work for me unless I have a big misunderstanding what needs to be done.
ALT+V is working and jumps to the next occurrence but ALT+A does nothing, except in the Markeren tab.
So what I'm doing is opening the dutch.xml
Make sure caret is on the first position.
CTRL+H
write `&amp;a`
ALT+V -> caret jumps to line 106
ALT+A -> nothing happens
goto Markeren tab
ALT+A -> all occurrences are marked

But to be honest, even if it would have worked, I don't think this is expected behavior.
If there is a shortcut it should react as long as the find dialog has the focus.
What I just did is replacing two of the three `&amp;a` instances to make each of them unique and it seems to do the job.

What am I missing?

**Comment by xylographe** [comment]
Very strange,
Ctrl-H
click Next ("Volgende zoeken")
Alt-A
 → all occurrences replaced
Esc
Ctrl-Z (undo Replace All)
Ctrl-H
Alt-V
Alt-A
 → nothing happens

**Comment by ThelsK** [comment]
I don't go out of my way to select specific boxes or anything, no. I know that when using the English version, it works just fine and dandy every time. It's the Dutch version that had problems for me.

**Comment by xylographe** [comment]
After replacing
```xml
<Item id="1615" name="&amp;Alle zoeken"/>
```
with
```xml
<Item id="1615" name="Alle &amp;markeren"/>
```
in `%APPDATA%\Notepad++\nativeLang.xml`, and restarting Notepad++,
Ctrl-H
Alt-A
 → all occurrences replaced

Hence, the bug that makes it impossible _to use the same shortcut for different tabs in the find dialogue_ has not been fixed! Thank you, @Ekopalypse.

Now I have to figure out what to do with **Alt-B** — can't use it in Search _and_ Replace tab.

**Comment by xylographe** [comment]
To try [fixed dutch.xml](https://raw.githubusercontent.com/notepad-plus-plus/notepad-plus-plus/2d31e75d7f3bc1aee7c7363e86d20d8441aaa6f8/PowerEditor/installer/nativeLang/dutch.xml),
download into `%APPDATA%\Notepad++`,
rename `nativeLang.xml` to `english.xml`,
rename `dutch.xml` to `nativeLang.xml`,
restart Notepad++.

**Comment by Ekopalypse** [comment]
Fixed one worked for me.

**Comment by wonkawilly** [comment]
Sorry for the OT, I'm also really sorry if I'm bothering you all, but maybe the participants to this thread would want to give their opinion about this new and rich proposal about the improvement of the Find and Replace dialog at this link: #9627. Which will be a great improvement and perhaps even a real game changer for all of us that use N++ for more advanced uses than just editing the shopping list.
Please partecipate and spread the word asking others to partecipate as well. More opinions are welcome. Thank you all.
